content/en/docs/about/community.md

```diff
@@ -5,30 +5,22 @@ weight = 10
 aliases = ["/docs/community/"]
 +++
 
-## Kubeflow Slack
-
-The Official Kubeflow Slack is used for informal discussions among users and contributors.
-
-<a href="https://communityinviter.com/apps/kubeflow/slack">
-  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br>Kubeflow Slack</button>
-</a>
-
-| Popular Kubeflow Slack Channels |
-| --- | --- |
-| General Discussion | [#general](https://kubeflow.slack.com/archives/C7REE0ETX)
-| Feature Requests | [#feature-requests](https://kubeflow.slack.com/archives/C01A7RYEYMB)
-| Job Postings | [#job-postings](https://kubeflow.slack.com/archives/CJ9PJE5FS)
-| Kubeflow - Pipelines | [#kubeflow-pipelines](https://kubeflow.slack.com/archives/CE10KS9M4)
-| Kubeflow - Notebooks | [#kubeflow-notebooks](https://kubeflow.slack.com/archives/CESP7FCQ7)
-| Kubeflow - Training Operator | [#kubeflow-training-operator](https://kubeflow.slack.com/archives/C985VJN9F)
-| Kubeflow - Katib | [#kubeflow-katib](https://kubeflow.slack.com/archives/CE0BURK1B)
-| KServe | [#kserve](https://kubeflow.slack.com/archives/CH6E58LNP)
-| Platform - AWS | [#platform-aws](https://kubeflow.slack.com/archives/CKBA5D0MU)
-| Platform - Azure | [#platform-azure](https://kubeflow.slack.com/archives/CUW6SLCPR)
-| Platform - GCP | [#platform-gcp](https://kubeflow.slack.com/archives/CKH7V1M7F)
-| Users - China | [#users-china](https://kubeflow.slack.com/archives/C93HYNM9C)
-| Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
-| Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
+## Kubeflow Slack Channels
+
+Kubeflow is part of the vibrant CNCF community. You are welcome to create your CNCF Slack account [here](https://slack.cncf.io/) and then join the following list of Kubeflow Slack channels.
+
```

Comment:
To help users understand that something has changed, we need to include a callout.
We can also link the old invitation here in a non-prominent way, to help users understand.

Here is my suggestion:

```suggestion

{{% alert title="Note" color="dark" %}}
The Kubeflow project migrated to the CNCF Slack in __July 2024__.
Historically, there was a dedicated [_Kubeflow Slack_](https://communityinviter.com/apps/kubeflow/slack) which is now deprecated.
{{% /alert %}}

```

Reply:
If people are violently against linking the old server, we can remove that.

But we absolutely need to reference the fact that something changed.

Reply:
We can link to KSC decision record https://github.com/kubeflow/community/blob/master/ksc/decision-log.md

I don't think we need to link to the old Slack.

Reply:
I still don't understand what is the user value to tell about the old Kubeflow Slack ?

All of the existing users are already in Kubeflow Slack, and they will be slowly migrated to the new one.
New users have to go directly to the new CNCF Slack.

Reply:
I am less concerned about the link to the old slack, and more concerned about having a notice that something changed on XXX date, just remove the link and make it:

```md
{{% alert title="Note" color="dark" %}}
The Kubeflow project migrated to the CNCF Slack in __July 2024__.
Historically, there was a dedicated _Kubeflow Slack_ which is now deprecated.
{{% /alert %}}
```

Reply:
Also, because the PR changed, the location I am proposing to add this is directly under the "join button", not under the "slack channels" section.

Reply:
Personal opinion - I don't think we should mention the old Slack at all. Existing users will know once we start notifying.

Reply:
Yes, from my point of view, there is no user value to show this warning/message.

Reply:
@andreyvelich @terrytangyuan the old Slack was used for 6+ years, it will help users migrate if they are aware that something has changed.

It's not a permanent notice, but it is critical during the next few transition months.

Silently changing things will confuse people.

_PS: If you disagree with the specific wording, please suggest another, I am not concerned with the exact wording of the notice._

Reply:
@terrytangyuan and I have different view on this. We should just link the new CNCF slack, similar to how Argo community migrated to the CNCF Slack. Their transition was successful given that `argo` channels are one of the biggest in CNCF Slack.

I don't see any user value to have this warning on the Kubeflow website.

Do we have anyone in the Kubeflow community who has different thoughts here ?

/assign @johnugeorge @jbottum @james-jwu @zijianjoy @StefanoFioravanzo @kubeflow/wg-pipeline-leads @kubeflow/wg-training-leads @kubeflow/wg-notebooks-leads @kubeflow/wg-manifests-leads @akgraner @hbelmiro  @kubeflow/wg-data-leads

Reply:
+1 to @andreyvelich  I don't see any advantage in referring to the previous slack channel

Reply:
During KSC, we decided to not mention previous Slack.